intl.emu

```diff
@@ -79,17 +77,13 @@
           1. Set _x_ to 0.
         1. <ins>If _x_ is a Decimal128 value, then</ins>
           1. <ins>Let _sign_ be *undefined*.</ins>
-          1. <ins>Assert: _x_ is not *NaN*<sub>𝔻</sub>.</ins>
-          1. <ins>Assert: _x_ is not *+∞*<sub>𝔻</sub>.</ins>
-          1. <ins>Assert: _x_ is not *-∞*<sub>𝔻</sub>.</ins>
-          1. <ins>Let _v_ be _x_[0].</ins>
+          1. <ins>Let _v_ be cohort(_x_).</ins>
           1. <ins>If _v_ is *-0*<sub>𝔻</sub>, then</ins>
-            1. <ins>Set _sign_ be ~negative~.</ins>
+            1. <ins>Set _sign_ to ~negative~.</ins>
```

Comment:
This is not something changed by this PR, but you don't need to pre-define `_sign_` as undefined, because all branches will declare it with `Let` (it's the preference in 262, I'm not sure about what is the convention in 402)

Reply:
done